src/ui/proxy/interceptors.go

```diff
@@ -207,9 +207,10 @@ func (vh vulnerableHandler) ServeHTTP(rw http.ResponseWriter, req *http.Request)
 		return
 	}
 	imageSev := overview.Sev
-	if imageSev > int(projectVulnerableSeverity) {
-		log.Debugf("the image severity is higher then project setting, failing the response.")
-		http.Error(rw, marshalError("The image scan result doesn't pass the project setting.", http.StatusPreconditionFailed), http.StatusPreconditionFailed)
+	if imageSev >= int(projectVulnerableSeverity) {
+		log.Debugf("the image severity: %q is higher then project setting: %q, failing the response.", models.Severity(imageSev), projectVulnerableSeverity)
+		http.Error(rw, marshalError(fmt.Sprintf("The image scan result: %q is not slower than the project setting: %q.", models.Severity(imageSev), projectVulnerableSeverity),
```

Comment:
Could you reword the message to:

"The severity of vulnerability of the image: %q is equal or higher than the threshold in project setting: %q."